lib/osv/tests.py

```diff
@@ -12,17 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Test helpers."""
+import datetime
 import os
 import requests
 import subprocess
+import tempfile
 import threading
+from unittest import mock
+
+import pygit2
 
 _EMULATOR_TIMEOUT = 30
 _DATASTORE_EMULATOR_PORT = 8002
 _DATASTORE_READY_INDICATOR = b'is now running'
 TEST_PROJECT_ID = 'test-osv'
 
 
+class MockRepo:
+  """Mock repo."""
+
+  def __init__(self, path):
+    self.path = path
+    self._repo = pygit2.init_repository(path, True)
+    tree = self._repo.TreeBuilder().write()
+    author = pygit2.Signature('OSV', 'infra@osv.dev')
+    self._repo.create_commit('HEAD', author, author, 'Initial commit', tree, [])
+
+  def add_file(self, path, contents):
+    """Add a file."""
```

Comment:
Super nit: I think the style guide says to make function docstrings in descriptive style ("Adds a file").

Reply:
Done and fixed every case in this file.